lib/kafo/kafo_configure.rb

```diff
@@ -375,6 +385,7 @@ def unset
 
     def config_file
       return CONFIG_FILE if defined?(CONFIG_FILE) && File.exists?(CONFIG_FILE)
+      return self.class.scenario_manager.select_scenario if self.class.scenario_manager.configured?
```

Comment:
this can cause fatal error if there's no scenario, unfortunately it applies even in `--help` and `-h` runs, since this is called early at https://github.com/theforeman/kafo/pull/117/files#diff-657b82e4fb9c80f11badef608b14de10L43